pagopa/io-app
# chore(IT Wallet): [SIW-1363] Add IT Wallet credential cards in wallet

## Short description
This PR adds the ability to show credentials inside the new wallet section.

## List of changes proposed in this pull request
- Updated `ItwCredentialCard`
- Added `WrappedItwCredentialCard`
- Updated DS screen

## How to test
Static checks should pass.
Go to **Profile > Human Interface > Design System > Wallet**, check that the cards are displayed correctly.

## Preview

https://github.com/user-attachments/assets/7c1acea9-5513-430f-a2cd-0d568f63d100
